speed-dreams/source-2.2.3/data/data/indicators/config.xml

```diff
@@ -70,5 +70,22 @@
                 <attstr name ="source" val="warning_brake_brake.wav"/>
             </section>
         </section>
+		<section name ="accelerate">
+			<section name="texture">
+				<attstr name ="source" val="Intervention_Brake_Proto_256.png"/>
```

Comment:
Maybe ask the artists whether they have a fitting texture? Adding a brake texture to the accel decision is not really a good idea.

Reply:
Hier maken we waarschijnlijk een aparte bug van in Jira

Reply:
Alright, remove the texture and sound data for now until the accel texture and sounds are available. Indicator will then only have a text displayed (for now).